lib/counter_culture/counter.rb

```diff
@@ -12,6 +12,7 @@ def initialize(model, relation, options)
       @counter_cache_name = options.fetch(:column_name, "#{model.name.tableize}_count")
       @column_names = options[:column_names]
       @delta_column = options[:delta_column]
+      @extra_join = options[:extra_join]
```

Comment:
do we need to call it `extra`? would it be confusing to call it simply `join`? 🤷‍♂️

Reply:
makes sense, to not make it extra confusing :trollface: